README.md

```diff
@@ -55,6 +55,7 @@ claims no expertise beyond English.
 | si     | Sinhala              | 4.0.3   | [npwsamarasinghe][si]| Nalaka Prasad
 | sk     | Slovak               | 4.0.0   | [xbotkaj][sk]        | Jozef Botka
 | sv-SE  | Swedish - Sweden     | 4.0.3   | [tangix][sv-SE]      | Mattias Sandström
+| th     | Thai                 | 4.0.3   | [karidoe][th]        | Anuchit Thiamuan
```

Comment:
Please change the version origin to 4.x to denote this is still unreleased. I'll change this later when we have a release.

Reply:
got it.